agents/go-agents/exec-agent/main.go

```diff
@@ -17,15 +17,18 @@ import (
 	"net/http"
 	"net/url"
 	"os"
+	"regexp"
 	"time"
 
+	"strconv"
```

Comment:
Should this import really be in a separate section? Maybe we can append it to previous?

Reply:
+1

Reply:
Is this `+1` to first or second comment? =)

Reply:
fixed